### PR TITLE
Borg Module QoL and Gas Analyzer Module Removal

### PR DESCRIPTION
**Pull request by FungiFellow**
## About the PR

Altered Content of Several Borg Modules for QoL:

Added Hand Radio to Mining Module 

Added GPS to Grapple Module 

Exchanged Service Module Syringe with a BorgDropper

Added Dropper to Advanced Cleaning Module

Added Geiger Counter, Gas Analyzer, and Remote Signaller to Advanced Tools Module

Removed Gas Analyzer Module 

## Why / Balance

I feel like these changes will make it easier for Borgs going forward without upping their capabilities too much
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image
image
image
image
image

## Breaking changes

**Changelog**

:cl:
- tweak: Altered content of several Borg Modules for QoL
- remove: Removed Gas Analyzer Module

### CODE REVIEW

**Comment by FungiFellow** [comment]
I had to fight my IDE to take these Screenshots.

**Comment by IProduceWidgets** [comment]
The the syringe change just so service can't inject people?

**Comment by mirrorcult** [comment]
test fails are real 
image

**Comment by FungiFellow** [comment]
"test fails are real"

Nuh Uh (Fixed it,)

**Comment by FungiFellow** [comment]
> The the syringe change just so service can't inject people?

I removed the syringe because a dropper works perfectly fine and I don't want a Budget Mediborg

**Comment by deltanedas** [comment]
dropper is 3x slower

**Comment by FungiFellow** [comment]
Is there an actual difference between the BorgDropper and Regular Dropper?

**Comment by FungiFellow** [comment]
Tested and the only difference is the BorgDropper can hold 10u and Dropper holds 5u

**Comment by FungiFellow** [comment]
I'm still switching out the syringe because service borgs don't need to be able to inject people

**Comment by Arendian** [comment]
Making the borg dropper have 15u capacity sounds like the perfect solution.

**Comment by FungiFellow** [comment]
Gonna Mark as Draft for now while I modify some things.

Also Arendian I saw https://github.com/space-wizards/space-station-14/pull/23696/commits/466a5ae3ed18738e5cdde6c62be0faf3f0ab2057 and I'm basically gonna copy this code for the Crew Monitor to resolve the battery issue

**Comment by metalgearsloth** [comment]
Powercreep is real.

Remove crew monitor at least

**Comment by FungiFellow** [comment]
Why do you feel it's powercreep?

File: Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml [hunk 1]
```diff
@@ -30,6 +30,21 @@
   - type: StaticPrice
     price: 500
 
+- type: entity
+  parent: HandheldCrewMonitor
+  id: HandheldCrewMonitorBorg
+  suffix: borg
+  components:
+  - type: HandheldCrewMonitor
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml:38]: borgs cant insert cells

Comment (FungiFellow) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml:38]: If I did it right it should have a Microreactor cell in it that cent be ejected or switched

Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml:38]: why change chargeuse too

Comment (FungiFellow) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml:38]: I need to have the consumption increased to outpace the recharge of the Microreactor cell so it's not constantly kept open.

**Comment by FungiFellow** [comment]
> Remove crew monitor at least

I have tripled the power use of the Borg Crew Monitor as a drawback

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Crew monitoring is not supposed to be mobile it needs removing.

**Comment by GreyMario** [comment]
> Crew monitoring is not supposed to be mobile it needs removing.

ok then get rid of the handheld crew monitor item

**Comment by FungiFellow** [comment]
> Crew monitoring is not supposed to be mobile 

Handheld Crew Monitor go BRR

**Comment by metalgearsloth** [comment]
> > Crew monitoring is not supposed to be mobile it needs removing.
> 
> ok then get rid of the handheld crew monitor item

It has no bearing on this pr but https://github.com/space-wizards/space-station-14/pull/24761

**Comment by FungiFellow** [comment]
> > > Crew monitoring is not supposed to be mobile it needs removing.
> > 
> > 
> > ok then get rid of the handheld crew monitor item
> 
> It has no bearing on this pr but #24761

In that case I will get rid of the Crew Monitor, please lead with saying Handhelds are getting phased out.

**Comment by FungiFellow** [comment]
I have addressed the requested Changes

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Remove construction stacks https://github.com/space-wizards/space-station-14/issues/24792 this is just a bandaid and I don't want to increase tech debt.

**Comment by FungiFellow** [comment]
> Remove construction stacks #24792 this is just a bandaid and I don't want to increase tech debt.
that's free hands with a) extra steps and b) no way to know the whitelist

Do you mean construction stacks for the Module as a Whole or just my proposed Plasteel and Reinforced Glass Changes?

**Comment by FungiFellow** [comment]
Whitelisted Free Hands requires More Code as a Whole and leads to more Tech Debt than adding a Lingering version of a Stack

**Comment by FungiFellow** [comment]
> > > Remove construction stacks #24792 this is just a bandaid and I don't want to increase tech debt.
> > > that's free hands with a) extra steps and b) no way to know the whitelist
> > 
> > 
> > Do you mean construction stacks for the Module as a Whole or just my proposed Plasteel and Reinforced Glass Changes?
> 
> For now just don't make the problem worse.

Done

**Comment by LankLTE** [comment]
Changelog should be modified to be more concise. I don't think every change to every module has to be included outright.

**Comment by FungiFellow** [comment]
> Changelog should be modified to be more concise. I don't think every change to every module has to be included outright.

Done

**Pull request merged**